transaction/core/src/ring_signature/rct_bulletproofs.rs

```diff
@@ -582,7 +557,7 @@ mod rct_bulletproofs_tests {
                 fee,
                 &mut rng,
             );
-            assert!(result.is_ok());
+            assert_eq!(result, Ok(()));
```

Comment:
Why make this change?  Do we have a preferred style for asserts?

Reply:
The advantage `assert_eq!()` has over `assert!` is that when the assert fails, `assert_eq` will show you the value it wasn't equal to, which makes debugging easier. `assert!(result.is_ok())` would fail, but you wouldn't see which Err caused it.